include/string_utils.h

```diff
@@ -103,6 +103,7 @@ bool starts_with(const char (&pfx)[N], const std::string &str) noexcept
 bool ends_with(const std::string &str, const std::string &suffix) noexcept;
 
 bool find_in_case_insensitive(const std::string &needle, const std::string &haystack);
+std::string::iterator ascii_to_lower(std::string &str);
```

Comment:
We've got the not-so-nicely named `lowcase(..)` functions in support (for char* and std::string), both do in-place lowering. Implementation is very close - but if yours is improved, feel free to swap it in.

Reply:
How did I miss that!

Probably because it was in support, and I was mostly looking in string_utils.h...

I'll use the existing function for now, improvements can be made later if desired.

Reply:
Now using existing `lowcase(...)`.